content/applications/general/users/access_rights.rst

```diff
@@ -52,6 +52,65 @@ The :guilabel:`Administration` field in the :guilabel:`Access Rights` tab has th
 .. image:: access_rights/user-permissions-dropdown-menu.png
    :alt: The Sales apps drop-down menu to set the user's level of permissions.
 
+Manage specific permissions
+---------------------------
+
+While access rights are typically assigned in bundles under specific roles, they can also be set as
+explicit permissions.
+
+.. example::
+   For example, giving a user the :guilabel:`Administrator` permission for **Timesheets**
+   gives them full access to that app. That user, while holding full access, can *still* have their
+   ability to manage *their own* timesheets restricted — such as in the case of a salaried payroll
+   administrator who does not need to track time.
+
+To manage specific permissions, :ref:`developer mode <developer-mode>` must be enabled. After that,
+navigate to the :menuselection:`Settings` app. Then click :guilabel:`Manage Users`, select a user,
+and go to the :guilabel:`Technical Access Rights` tab. From here, :guilabel:`Groups` can be edited,
+and specific access rights can be managed across the various sections. If no changes are made to
+these *Groups*, then their permissions will mirror the selections made in the :guilabel:`Access
+Rights` tab.
+
+- :guilabel:`Selected groups`: a list of detailed access rights, set by choices made in the
+  :guilabel:`Access Rights` tab.
+- :guilabel:`Groups added automatically`: *implied* permissions that are *inherited* with the
+  explicit permissions already granted to the user. The values here will match the values listed
+  under a given *Group*'s form located under the :menuselection:` Users & Companies --> Groups`
+  menu, in the :guilabel:`Inherited` tab.
```

Comment:
No need to add formatting to "group's", fix menuselection formatting, and add the app to beginning of the menu for clarity of instruction
```suggestion
  under a given group's form located under the :menuselection:`Settings --> Users & Companies -->
  Groups` menu, in the :guilabel:`Inherited` tab.
```

Reply:
I wrote it this way on purpose so I'd say this suggestion is optional @justinmallette 
- *Group* is referencing a section on the UI / Odoo feature so italicized like this is also fine
- Adding Settings in the menuselection is fine however we're already in the Settings at this point so can just point to the menu that should be present anyway.